cmd/pack/main.go

```diff
@@ -45,14 +45,19 @@ func main() {
 				if flag, err := fs.GetBool("timestamps"); err == nil {
 					logger.WantTime(flag)
 				}
+				if flag, err := fs.GetBool("experimental"); err == nil {
+					experimental = flag
+				}
 			}
 
 			packClient = initClient(logger)
+			packClient.Experimental(experimental)
```

Comment:
I don't see that this is persisted anywhere... would the user need to pass `--experimental` every time? This seems contradictory to the idea that we could see the current configuration by running `pack report`.